Gemfile

```diff
@@ -86,7 +86,7 @@ group :amazon, :manageiq_default do
 end
 
 group :ansible, :manageiq_default do
-  gem "ansible_tower_client",           "~>0.13.0",      :require => false
+  gem "ansible_tower_client",           "~>0.14.0",      :require => false
```

Comment:
we can remove this block because of https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/84 and the `manageiq-providers-ansible_tower` is already a dependency.

Reply:
I extracted that to a [separate PR](https://github.com/ManageIQ/manageiq/pull/17515).